src/test/java/gov/cms/mat/cql_elm_translation/config/logging/RequestResponseLoggingInterceptorTest.java

```diff
@@ -26,26 +26,21 @@ class RequestResponseLoggingInterceptorTest {
   @Mock private ClientHttpRequestExecution mockClientHttpRequestExecution;
 
   @Test
-  void test() {
+  void testHttpRequestIntercept() throws IOException {
```

Comment:
to fix the Jenkins build that failed previously. https://semanticbits.slack.com/archives/CNU5AEYMV/p1712266682960979
I'm not sure how useful is this interceptor because it only works if the log level is DEBUG .